relay-dynamic-config/src/defaults.rs

```diff
@@ -143,6 +147,63 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
                 })
                 .into(),
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "c:spans/data.llm.llm_completion_tkens@none".into(),
+            field: Some("span.data.llm_completion_tkens".into()),
+            condition: Some(RuleCondition::eq("span.op", "ai.llm.completion")),
+            tags: [
+                ("span.sentry_tags.", "group"),
+                ("span.sentry_tags.", "op"),
+                ("span.data.", "language_model"),
```

Comment:
Do you want to be able to query `language_model` from the spans dataset as well as from metrics? In that case, you'd have to add it to `sentry_tags` in [tag_extraction.rs](https://github.com/getsentry/relay/blob/19ab9c3d4990963a1dd065f41517426b5f07232d/relay-event-normalization/src/normalize/span/tag_extraction.rs#L1).